include/slot_machine.h

```diff
@@ -1,6 +1,25 @@
 #ifndef GUARD_SLOT_MACHINE_H
 #define GUARD_SLOT_MACHINE_H
 
+#define NUM_REELS         3
+#define REEL_NUM_TAGS     21
+#define REEL_TAG_HEIGHT   24
+
+// Lucky Flags
+#define LUCKY_BIAS_REPLAY     (1 << 0)
+#define LUCKY_BIAS_CHERRY     (1 << 1)
+#define LUCKY_BIAS_LOTAD      (1 << 2)
+#define LUCKY_BIAS_AZURILL    (1 << 3)
+#define LUCKY_BIAS_POWER      (1 << 4)
+#define LUCKY_BIAS_REELTIME   (1 << 5)
+#define LUCKY_BIAS_MIXED_777  (1 << 6)
+#define LUCKY_BIAS_777        (1 << 7)
+
+// Rows
+#define SLOT_TOP_ROW          0x0001
```

Comment:
It doesn't look like these 3 constants are used anywhere, unless my ctrl+F is failing me.  If they are used, consider removing the hexadecimal formatting for their values.  Otherwise, just remove them if they're not used.